호텔 예약 조회 및 상세조회 기능 추가

nasaoreo opened the pull request:
## 📌 PR 목적 
호텔 예약 조회 및 상세조회 기능을 추가한 것에 대한 코드 리뷰를 받기 위해 PR을 작성합니다.

## 👩‍💻 요구 사항과 구현 내용
- 로그인된 사용자의 호텔 예약 전체 기록을 조회하는 기능 추가
- 선택된 호텔 예약의 상세 내용을 조회하는 기능 추가

## ✅ 피드백 반영사항
<!-- 여기엔 피드백 반영사항을 작성해주세요 !! -->


## 🤔 PR 포인트 & 궁금한 점
<!-- 여기엔 PR 포인트 & 궁금한 점을 작성해주세요 !! -->
호텔 예약 조회 및 상세 조회 결과는 모두 HotelBookDetailsDto 타입으로 반환하였습니다. 호텔 예약 정보를 어느 정도 수준으로 간단하게 또는 복잡하게 보여줄 지는 프론트엔드와 협의하여 결정할 사항이라고 생각되기 때문에, 동일한 Dto 타입을 사용하였습니다~!

## 관련 이슈 
<!-- 예시) Close: #이슈 번호 -->
Closes #24
Closes #25

nasaoreo committed:
Refactor: HotelBookingResponseDto 필드명 변경

nasaoreo committed:
Refactor: HotelBookingResponseDto 필드명 변경

nasaoreo committed:
Refactor: Dto 클래스명 및 HotelService 메소드명 변경

- HotelRoomDto를 HotelRoomDetailsDto로 변경
- 메소드 목적을 더 명확하게 전달하기 위해 HotelService의 메소드명 변경

nasaoreo committed:
Refactor: Dto 클래스명 및 HotelService 메소드명 변경

- HotelRoomDto를 HotelRoomDetailsDto로 변경
- 메소드 목적을 더 명확하게 전달하기 위해 HotelService의 메소드명 변경

nasaoreo committed:
Feat: HotelBooking 엔티티에 price 컬럼 추가

- 개별 방에 가격을 나타내는 price 컬럼 추가
- HotelBookingService 리팩토링(메소드로 코드 분리)

nasaoreo committed:
Feat: HotelBooking 엔티티에 price 컬럼 추가

- 개별 방에 가격을 나타내는 price 컬럼 추가
- HotelBookingService 리팩토링(메소드로 코드 분리)

nasaoreo committed:
Feat: 호텔 예약 상세조회 기능 추가

nasaoreo committed:
Feat: 호텔 예약 상세조회 기능 추가

nasaoreo committed:
Feat: 호텔 예약 조회 기능 추가

nasaoreo committed:
Feat: 호텔 예약 조회 기능 추가

taehyunnn reviewed (approved):
오우. merge 오토 콜입니다~ 👍

dia312 reviewed (approved):
줄리~! 수고하셨씁니당 ㅎㅎ approve 하겠습니다!!

taehyunnn committed:
Feat: [#46] 예외 메시지 한글로 관리 및 로그 남기기 (#63)

- 예외 패키지 통합
- ExceptionMessageUtils 구현
- exception.properties 에서 모든 메시지 관리
- NotFoundException 은 하나로 통일
- 로그 레벨은 사용자에 관한건 warn 서버에 관한건 error

nasaoreo committed:
Fix: 예외처리 방식 변경에 따라 예외 관련 코드 전체 수정

nasaoreo committed:
Merge branch 'feat/issue-24' of github.com:CarrierRoute/BEDV1_CarrierRoute into feat/issue-24

# Conflicts:
#	src/main/java/com/grepp/carrierroute/booking/domain/HotelBooking.java
#	src/main/java/com/grepp/carrierroute/booking/service/HotelBookingService.java
#	src/main/java/com/grepp/carrierroute/hotel/exception/ErrorMessage.java
#	src/main/java/com/grepp/carrierroute/hotel/service/HotelService.java
